starter-project-backend-g32/src/controllers/comment.controller.ts

```diff
@@ -0,0 +1,95 @@
+import {Request , Response } from "express";
+const Comment = require('../models/comment');
```

Comment:
better to use 'import' statement to stick to Es6 importing and consistent importing scheme